### PR TITLE
Allow insertions with neither static columns nor timestamps.

### LINKED ISSUE #17: Must provide static_columns or turn on timestamps
If you provide an empty hash for `static_columns` and don't turn on the `timestamps` option:
- `column_names` will return `", variable_col1, variable_col2, etc"`.
- `insert_values` will add a `NULL` as the first value for each record.

### PR DESCRIPTION
**Pull request by patrickarnett**
Fixes #17

### CODE REVIEW

**Comment by sgringwe** [comment]
If you merge master, the sqlite errors are now fixed

**Comment by patrickarnett-ft** [comment]
This will fix #7, as well.

File: spec/fast_inserter_base_spec.rb [hunk 1]
```diff
@@ -180,6 +180,119 @@
...
+              expect(attendee.attendable_type).to eq attendable_types[i]
+              expect(attendee.checked_in).to eq checked_in[i]
+              expect(attendee.registered).to eq registered[i]
+            end
+          end
+        end
+      end
+
+      context "when timestamp option is not set" do
+        let(:created_at) { 4.times.map { |i| Time.now + Rational(10 * i, 86400) } }
```
Comment (sgringwe) [inline spec/fast_inserter_base_spec.rb]: the `Time.now + Rational(10 * i, 86400)` i found hard to decipher - could we instead use rspec syntax such as `expect(attendee.updated_at).to eq be_within(5.seconds).of(Time.current)`?

Comment (patrickarnett-ft) [reply spec/fast_inserter_base_spec.rb]: The `timestamps` option is already tested at `spec/fast_inserter_base_spec.rb:129`. My intention here is to confirm that the correct `values` are inserted into their respective `variable_columns`; two of those columns just happen to be the timestamp fields.

Comment (patrickarnett-ft) [reply spec/fast_inserter_base_spec.rb]: Then again, there are no explicit tests of the `timestamps` option without `static_columns`.

**Pull request closed without merging**